test/e2e/nns_conditions_test.go

```diff
@@ -56,6 +56,12 @@ var _ = Describe("NodeNetworkStateCondition", func() {
 		BeforeEach(func() {
 			updateDesiredState(invalidConfig)
 		})
+		AfterEach(func() {
```

Comment:
This should not be needed. If it is (due to a bug), then please explain in an inline TODO comment and link the bug.

Reply:
It fixes nothing I will rmeove it.